curator/controllers/study.py

```diff
@@ -100,7 +100,11 @@ def _get_latest_synthesis_details_for_study_id( study_id ):
             # Prepend scheme to a scheme-relative URL
             fetch_url = "https:%s" % fetch_url
         # as usual, this needs to be a POST (pass empty fetch_args)
-        source_list_response = requests.post(fetch_url, headers={"Content-Type": "application/json"},data=simplejson.dumps({'include_source_list':True})).text
+        source_list_response = requests.post(
+            fetch_url,
+            headers={"Content-Type": "application/json"},
+            data=simplejson.dumps({'include_source_list':True})
```

Comment:
Is it possible to switch from simplejson to json here?

Reply:
Certainly. I'll make that change in all three instances and update the PR. Thanks for looking out!